package.json

```diff
@@ -8,8 +8,10 @@
   },
   "license": "MIT",
   "dependencies": {
+    "crypto": "^1.0.1",
     "guid-typescript": "^1.0.9",
     "jsonwebtoken": "^8.5.1",
+    "oauth-1.0a": "^2.2.6",
```

Comment:
We use that only in the tests correct? Then it is a dev-dependency

Reply:
implemented